src/aat/resources/cucumber/questions.feature

```diff
@@ -35,6 +35,22 @@ Feature: Questions feature
     Then the response code is 200
     And the response contains 2 questions
 
+  Scenario: Retrieve all questions
+    Given a standard online hearing is created
+    And a standard question
+    And the question round is ' "1" '
+    And the post request is sent to create the question
+    Then the response code is 201
+    When the put request is sent to issue the question round ' "1" '
+    Then wait until the event is processed
+    And the response code is 200
+    And a standard answer
+    And the endpoint is for submitting an answer
+    And a POST request is sent
+    Then the response code is 201
+    When the get request is sent to retrieve all questions
+    And the response contains the following text '"answer_text" '
```

Comment:
This is fine in principle. It would be nice if the test could prove that the questions contained an array of answers e.g.
And the first question contains 1 answer 
where the cucumber step will test the size of the array.